config/routes.rb

```diff
@@ -115,6 +115,8 @@
 
     resources :sar_internal_review, only: only, controller: 'sar_internal_review', as: :case_sar_internal_review do
       get '/(:step)', on: :collection, to: 'sar_internal_review#new', as: 'step'
+      get '/edit', on: :member, to: 'offender_sar_complaint#edit' 
+      post '/update', on: :member, to: 'offender_sar_complaint#update'
```

Comment:
hmm... should it be `sar_internal_review#edit `and `sar_internal_review#update`?

Reply:
Yes 🤦